### PR TITLE
[sse2neon] Add new port

### PR DESCRIPTION
**Pull request by rami-lv**
**Describe the pull request**

- #### What does your PR fix?

Add a new port

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  all , NO

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes

### CODE REVIEW

File: ports/sse2neon/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    REF v1.5.1
+    SHA512  7d14b074aa5829e218a04975d96fc67adba3fb4ffd9d580ab0bc05048f7abd9143dd308b2d9e8b1600b00427b21b24d49d7a6670c91270fcc6d053c3bca908af
+    HEAD_REF master
+)
+
+# Copy header file
+file(COPY ${SOURCE_PATH}/sse2neon.h DESTINATION ${CURRENT_PACKAGES_DIR}/include/sse2neon/sse2neon.h)
+
+# Handle copyright
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```
Comment (Cheney-W) [inline ports/sse2neon/portfile.cmake:14]:
```suggestion
file(COPY "${SOURCE_PATH}/sse2neon.h" DESTINATION "${CURRENT_PACKAGES_DIR}/include/sse2neon/sse2neon.h")

# Handle copyright
vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
```

**Comment by Cheney-W** [comment]
Please sign the CLA according to the robot's instructions.

**Comment by Cheney-W** [comment]
This is the requirement to submit PR for Microsoft repository. It has nothing to do with that port.

**Comment by rami-lv** [comment]
@microsoft-github-policy-service agree

**Comment by Cheney-W** [comment]
Could you please apply my suggestion? Thank you!

**Comment by rami-lv** [comment]
> Could you please apply my suggestion? Thank you!

I believe I had to run `./vcpkg x-add-version --all` again right?

**Review by Cheney-W — APPROVED** [review]

**Pull request merged**